<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostNAS_SearchDialogBase</class>
 <widget class="QDialog" name="PostNAS_SearchDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KU_Search</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="showButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Auswahl anzeigen</string>
     </property>
     <property name="text">
      <string>Anzeigen</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/plugins/PostNAS_Search/search_16x16.png</normaloff>:/plugins/PostNAS_Search/search_16x16.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="resetButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Ergebnis l√∂schen</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/plugins/PostNAS_Search/marker-delete.png</normaloff>:/plugins/PostNAS_Search/marker-delete.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
